src/components/Sidebar/Sidebar.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { GU, useTheme, Link } from '@1hive/1hive-ui'
+import gardensLogo from '@assets/gardensLogoMark.svg'
+import MenuItem from './MenuItem'
+
+function Sidebar({ gardens, activeGarden, onOpen }) {
+  const theme = useTheme()
+  return (
+    <div
+      css={`
+        width: ${9 * GU}px;
+        flex-shrink: 0;
+        background: ${theme.surface};
+        border-right: 1px solid ${theme.border};
+        box-shadow: 2px 0 3px rgba(0, 0, 0, 0.05);
+      `}
+    >
+      <div
+        css={`
+          padding: ${2 * GU}px;
+          padding-bottom: ${1 * GU}px;
+        `}
+      >
+        <Link href="#/home" external={false}>
+          <div
+            css={`
+              border-bottom: 1px solid ${theme.border};
+            `}
+          >
+            <img src={gardensLogo} height={40} alt="" />
+          </div>
+        </Link>
+      </div>
```

Comment:
The spacing between the logo and the bottom line here looks a bit off:
<img width="93" alt="Screen Shot 2021-08-30 at 17 58 32" src="https://user-images.githubusercontent.com/22663232/131416405-be47fcb4-466c-4f3e-bfa6-84489df366fb.png">

Also i think the bottom line shouldn´t be part of the link to the gardens main page.

We can put the div outside and adjusting the spaces would loook something like

```suggestion
     <div
        css={`
          padding: ${1.5 * GU}px ${2 * GU}px;
        `}
      >
        <div
          css={`
            display: flex;
            padding-bottom: ${1.5 * GU}px;
            border-bottom: 1px solid ${theme.border};
          `}
        >
          <Link
            href="#/home"
            external={false}
            css={`
              display: block;
            `}
          >
            <img
              src={gardensLogo}
              height={40}
              alt=""
              css={`
                display: block;
              `}
            />
          </Link>
        </div>
      </div>
```